### DISCUSSION TITLE
ridiculouslab

### DISCUSSION THREAD

**Discussion by ridiculouslab** [philibertc/micronova_controller#66] [Show and tell]
Sorry, could you please update the RiduculousLab reference in your repository Micronova_controller for Micronova protocol analysis with the following link
https://www.ridiculouslab.com/arguments/iot/stufa/micronova_en.php
Tanks for your Job Philibert, merci beaucoup. 
Salut

**Comment by philibertc** [comment]
Hello, I only saw your comment this day. I updated the link.

Bests
